go/tools/gopackagesdriver/aspect.bzl

```diff
@@ -84,7 +89,13 @@ def _go_pkg_info_aspect_impl(target, ctx):
     transitive_compiled_go_files = []
 
     for attr in DEPS_ATTRS + PROTO_COMPILER_ATTRS:
-        for dep in getattr(ctx.rule.attr, attr, []) or []:
+        deps = getattr(ctx.rule.attr, attr, []) or []
+
+        # Some attrs are not iterable, ensure that deps is always iterable.
+        if type(deps) != "list":
```

Comment:
Could you replace that with a constant for `type([])`? The returned strings aren't guaranteed to be stable.

Reply:
Well that makes sense and is way more readable!